### PR TITLE
Refactor to use country lib instead of enum

### PR DESCRIPTION
**Pull request by qqTYXn7**

### CODE REVIEW

File: src/api/commands/shippingdestination/ShippingDestinationAddCommand.ts.orig [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            shippingDestination = await this.shippingDestinationService.create({
+                item_information_id: itemInformation.id,
+                country: countryCode,
+                shippingAvailability: shippingAvail
+=======
+            shippingDestination = await this.shippingDestinationService.create({ item_information_id: itemInformation.id,
+                country: Country[data.params[1]],
+                shippingAvailability: ShippingAvailability[data.params[2]]
+>>>>>>> f174ba52258265b0e4996f74b3d2dcd15d293728
+            } as ShippingDestinationCreateRequest);
```
Comment (xludx) [inline src/api/commands/shippingdestination/ShippingDestinationAddCommand.ts.orig]: merge conflict

File: src/api/enums/Country.ts [hunk 2]
```diff
@@ -1,16 +1,9 @@
...
 export enum Country {
-
     EU = 'EU',
     USA = 'USA',
     ASIA = 'ASIA',
     FINLAND = 'FINLAND',
     SWEDEN = 'SWEDEN',
     SOUTH_AFRICA = 'SOUTH_AFRICA',
     UNITED_KINGDOM = 'UNITED_KINGDOM'
-
```
Comment (xludx) [inline src/api/enums/Country.ts]: you can remove this enum once it's not used anymore.

File: src/api/commands/shippingdestination/ShippingDestinationAddCommand.ts.orig (hunk 1 above)
Comment (xludx) [reply src/api/commands/shippingdestination/ShippingDestinationAddCommand.ts.orig]: I saw the file name, you can remove this.

**Comment by qqTYXn7** [comment]
Unnecessary files were deleted

**Pull request merged**